internal/pkg/testutils/utils.go

```diff
@@ -584,12 +581,11 @@ func AddNode(c client.Interface, kc *kubernetes.Clientset, host string) error {
 }
 
 func DeleteNode(c client.Interface, kc *kubernetes.Clientset, host string) error {
-	var err error
-	err = nil
+	var err error = nil
```

Comment:
I don't see us ever using this - shouldn't we just change line 596 to `return nil`?

Reply:
We still use `err` on line 593 and return that value.